include/ibm/management_console_rest.hpp

```diff
@@ -1260,6 +1315,15 @@ inline void requestRoutes(App& app)
                const std::shared_ptr<bmcweb::AsyncResp>& asyncResp) {
         handleBroadcastService(req, asyncResp);
         });
+
+    BMCWEB_ROUTE(app, "/ibm/v1/OCC/Control/<str>/Actions/PassThrough.Send")
+        .privileges({{"ConfigureManager"}})
```

Comment:
Can we do         .privileges({{"OemIBMPerformService"}}) here like it is today?

https://github.com/ibm-openbmc/bmcweb/blob/1030/include/ibm/management_console_rest.hpp#L1351

Reply:
> Can we do .privileges({{"OemIBMPerformService"}}) here like it is today?
> 
> https://github.com/ibm-openbmc/bmcweb/blob/1030/include/ibm/management_console_rest.hpp#L1351

Done